src/xenia/memory.cc

```diff
@@ -353,6 +353,27 @@ BaseHeap* Memory::LookupHeapByType(bool physical, uint32_t page_size) {
 
 VirtualHeap* Memory::GetPhysicalHeap() { return &heaps_.physical; }
 
+void Memory::GetHeapsPageStatsSummary(const BaseHeap* provided_heaps[],
```

Comment:
`const BaseHeap* const* provided_heaps`, as both the heaps themselves and the array of the heaps are not modified by this function.